tron/utils/scribereader.py

```diff
@@ -25,45 +26,12 @@
 USE_SRV_CONFIGS = -1
 
 
-@lru_cache(maxsize=1)
-def get_region() -> str:
-    """
-    Discover what region we're running in by reading this information from on-disk facts.
-
-    Yelpers: for more information, see y/habitat
-    """
-    with open("/nail/etc/region") as f:
-        return f.read().strip()
-
-
-@lru_cache(maxsize=1)
-def get_superregion() -> str:
```

Comment:
This mock.patch was not updated: https://github.com/Yelp/Tron/blob/4da0f48/tests/utils/scribereader_test.py#L27